system/lib/libcxx/readme.txt

```diff
@@ -1,7 +1,7 @@
-These files are from libc++, release 9.0.0.
+These files are from libc++, release branch 10.x.
 
-tag: llvmorg-9.0.0
-git: 0399d5a9682b3cef71c653373e38890c63c4c365
+tag: none
```

Comment:
Is there no tag that you sync'd to?  Wouldn't it be better to sync to a given tag?

Reply:
There is separate readme for libcxxabi

Reply:
The current revision is past the 10.0 release tag, but since all of the commits on the release branch are bugfixes and such, I didn't really see any reason to leave any out?
Fixed the libcxxabi readme in the next commit.

Reply:
hmm.. still they are as-yet-unreleased changes.    Why not just stick with the tag.   No need to go back and change now, we can aim for tag next time.

Reply:
Turns out none of the changes on the branch touched libcxx or libcxxabi, we can just pretend it came from the tagged revision.